### PR TITLE
Home explore profile bookmark UI and routing

### PR DESCRIPTION
**Pull request by mikesch803**
1. add home-page
2. add explore-page
3. add profile-page
4. add bookmark-page
5. add 404
6. basic routing between above pages **(no functionality)**

### CODE REVIEW

File: src/context/post-context.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  const [post, setPost] = useState([]);
+
+  useEffect(() => {
+    (async () => {
+      try {
+        const response = await axios.get("/api/posts");
+        if (response.status === 200) {
+          setPost(response.data.posts);
+        }
+        console.log(response);
```
Comment (ApurvaSawant11) [inline src/context/post-context.jsx:16]: You can remove the console.log() if not needed

File: src/components/aside/Aside.css [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    padding: 0.5rem;
+    background-color: var(--COLOR-FT);
+}
+
+.follow-user {
+    display: flex;
+    /* padding: 0.5rem; */
+    align-items: center;
+    
+    /* border: 1px solid #e0dede; */
```
Comment (ApurvaSawant11) [inline src/components/aside/Aside.css:35]: Remove all the comments from this file and fix the indentation

**Review by ApurvaSawant11 — APPROVED** [review]
Looks good to me. Did not find anything to suggest. You can merge the PR

**Review by hsnice16 — APPROVED** [review]
Great work, man. Couldn't find anything to suggest. Good Job.

**Pull request merged**